training-event/src/main/java/org/example/config/JMSConfig.java

```diff
@@ -18,6 +20,9 @@ public class JMSConfig {
     @Value("${spring.activemq.broker-url}")
     private String brokerUrl;
 
+    @Autowired
+    private ObjectMapper objectMapper;
```

Comment:
this bean is used only once, better to inject it to method itself instead of having field on config class level. Also, prefer to have `final` fields and inject them via constructor